packages/design-system/src/components/AvatarGroup/index.tsx

```diff
@@ -13,14 +13,26 @@ export const AvatarGroup: React.FC<AvatarGroupProps> = ({
   const visibleAvatarsArray = avatars.slice(0, visibleAvatars);
 
   return (
-    <div className={classNames("flex -space-x-3", className)}>
+    <div
+      className={classNames(
+        "flex",
+        className,
+        (size === "xxs" || size === "xs") && "-space-x-2",
+        (size === "sm" || size === "md") && "-space-x-3.5",
+        (size === "lg" || size === "xl" || size === "xxl") && "-space-x-3"
+      )}
+    >
       {visibleAvatarsArray.map((avatar, index) => (
         <Avatar
           {...avatar}
           key={`${index}-id`}
           size={size}
           shape={shape}
           contentType={contentType}
+          className={classNames(
+            "border-avatarGroup-base",
+            size === "xxs" || size === "xs" ? "border-[1.5px]" : "border-2"
```

Comment:
What about create `const extraSmallVariants = size === "xxs" || size === "xs"`
```suggestion
            extraSmallVariants ? "border-[1.5px]" : "border-2"
```

or `const borderWidthVariants = size === "xxs" || size === "xs" ? "border-[1.5px]" : "border-2"`
```suggestion
           borderWidthVariants
```